src/components/molecules/Alert/Alert.test.tsx

```diff
@@ -2,56 +2,34 @@ import { composeStories } from '@storybook/react';
 import { render, screen } from 'src/test-utils.js';
 import * as stories from './Alert.stories';
 
-const { Default, WithCloseButton } = composeStories(stories);
+const { Info, Error } = composeStories(stories);
 
 describe('Alert Component', () => {
-    it('should render a Alert', () => {
-        render(<Default />);
+    it('should render an Alert', () => {
+        render(<Info />);
     });
 
-    it('should render solid variant if variant is not provided', () => {
-        render(<Default />);
+    it('should render info variant if variant is not provided', () => {
+        render(<Info />);
         const alert = screen.getByRole('alert');
-        expect(alert).toHaveClass(
-            'bg-[rgba(41, 45, 63, 0.60)] border border-white-10 shadow-tab'
-        );
-        expect(alert).not.toHaveClass('border-2 border-yellow bg-yellow/20');
+        expect(alert).toHaveClass('border-primary-300 bg-primary-500/10');
     });
 
-    it('should render outlined variant if specified', () => {
-        render(<Default variant='outlined' />);
+    it('should render error variant if specified', () => {
+        render(<Error />);
         const alert = screen.getByRole('alert');
-        expect(alert).not.toHaveClass(
-            'bg-[rgba(41, 45, 63, 0.60)] border border-white-10 shadow-tab'
-        );
-        expect(alert).toHaveClass('border-2 border-yellow bg-yellow/20');
+        expect(alert).toHaveClass('border-error-300 bg-error-500/10');
     });
 
-    it('should not render a close button if showCloseButton is false', () => {
-        render(<Default />);
-        const closeButton = screen.queryByTestId('close-button');
-        expect(closeButton).not.toBeInTheDocument();
-    });
-
-    it('should  render a close button if showCloseButton is true', () => {
```

Comment:
the closeButton test should still be here as the button is always present now.

Reply:
Got it. I also updated the test for functionality on clicking the button